lib/runner/TestRunner.js

```diff
@@ -49,10 +49,20 @@ module.exports = class TestRunner {
         const locales = testSuite.locales;
         if (locales) {
             const localesList = locales.split(",").map(l => l.trim());
-            let results = await Promise.all(localesList.map((locale) => {
-                return this.runSuiteForLocale(testSuite, context, locale);
-            }));
+            const runInBand = this.runInBand();
+            let results;
+            if (runInBand) {
```

Comment:
I would run all of the tests sequentially, whether it is unit or e2e. It is safer to run it this way, even if it is a bit slower.